app/controllers/employers_controller.rb

```diff
@@ -10,6 +11,7 @@ def index
   # GET /employers/1
   # GET /employers/1.json
   def show
+    @employer = Employer.find_by(params[:id])
```

Comment:
giải thích giúp em sự khác nhau giữa find và find_by

Reply:
theo anh hiểu:
find(1) thì tìm mặc định theo id (=1)
còn find_by thì tìm theo nhiều thông số hơn, ví dụ id, name, sđt v.v...

Reply:
Ngoài ra, nếu ko tìm thấy thì find sẽ đưa về 1 exception, có thể là nguyên nhân dẫn đến crash server còn find_by trả về nil
`>> User.create name: 'THE USER' # creates a user with id = 1
>> User.find(1) # returns the user
>> User.find_by(id: 1) # returns the user

>> User.find(2) # raises an exception
>> User.find_by(id: 2) # returns nil`

Reply:
(y) chuẩn.. :D